docs/testrunner-toolkit/running-tests.md

```diff
@@ -28,7 +28,7 @@ Unless you specify a test directory, `saucectl` executes tests based on the test
 
 The following steps outline how to run your tests on your local machine with the containerized solution. This is the preferred option if you wish to run tests on your local machine, or if you wish to accelerate test execution in CI.
 
-Please note that if you don't specify a `--test-env` flag when running tests, `saucectl` defaults to `sauce` mode.
+Please note that if you don't specify a `--test-env` flag when running tests, `saucectl` defaults to `mode` settings in saucectl config file.
```

Comment:
So I think we need to rewrite this whole section to direct users to use the config file to specify the `mode` for the project as `docker` rather than using the CLI flag.